lib/ui/painting/image_decoder.cc

```diff
@@ -105,7 +105,8 @@ sk_sp<SkImage> ImageFromCompressedData(fml::RefPtr<ImageDescriptor> descriptor,
 
   if (!descriptor->should_resize(target_width, target_height)) {
     // No resizing requested. Just decode & rasterize the image.
-    return descriptor->image()->makeRasterImage();
+    sk_sp<SkImage> image = descriptor->image();
+    return image ? image->makeRasterImage() : nullptr;
```

Comment:
The pattern in this file seems to be to log errors before returning `nullptr`.

Reply:
There is no log in the other null `descriptor->image()` code path in this function
(https://github.com/flutter/engine/blob/master/lib/ui/painting/image_decoder.cc#L154)

`ImageDescriptor::image()` will log the cause if it returns null.